lib/stack_master/commands/apply.rb

```diff
@@ -46,11 +46,19 @@ def stack_exists?
         !stack.nil?
       end
 
+      def abort_if_review_in_progress
+        if stack_exists? && stack.stack_status == "REVIEW_IN_PROGRESS"
```

Comment:
I don't think `REVIEW_IN_PROGRESS` is necessarily a bad state? Do we get in this state if we propose a change and select `n` to applying it?

The main problem we encounter is an empty stack

Reply:
We do cleanup if you ctrl c now, but can't if you there are no permissions. I'm just wondering that instead of checking for the status, should we check for an empty stack instead?

Reply:
hmm here is the result of `stack.inspect` when we get this issue.

```
#<StackMaster::Stack:0x00007fd95d50b070 @region="us-east-1", @stack_name="mistack", @stack_id="arn:aws:cloudformation:us-east-1:123456789:stack/mistack-database/6c029cb0-5901-11e8-a5fa-500c2854b699", @parameters={}, @template_body="", @template_format=:yaml, @outputs=[], @role_arn=nil, @notification_arns=[], @stack_policy_body=nil, @stack_status="REVIEW_IN_PROGRESS">
```
do you think we can test on template_body?